test/support/models.rb

```diff
@@ -128,29 +128,58 @@ def column_for_attribute(attribute)
 
   begin
     require 'active_model/type'
-    def type_for_attribute(attribute)
-      column_type, limit = case attribute
-        when 'name', 'status', 'password' then [:string, 100]
-        when 'description'   then [:string, 200] # because :text is database-adapter specific
-        when 'age'           then :integer
-        when 'credit_limit'  then [:decimal, 15]
-        when 'active'        then :boolean
-        when 'born_at'       then :date
-        when 'delivery_time' then :time
-        when 'created_at'    then :datetime
-        when 'updated_at'    then :datetime
-        when 'lock_version'  then :integer
-        when 'home_picture'  then :string
-        when 'amount'        then :integer
-        when 'attempts'      then :integer
-        when 'action'        then :string
-        when 'credit_card'   then :string
-        when 'uuid'          then :string
-      end
 
-      ActiveModel::Type.lookup(column_type, limit: limit)
+    begin
```

Comment:
Why not create a Hash first that maps all the types and then only overwrite that one specific entry?